### PR TITLE
Add eip1559 params in send/sendMany

### PR DESCRIPTION
**Pull request by karankurbur**
Adds eip1559 params to send/sendMany routes. If we pass in the new params it will pass it /build and insert the new params as part of the halfSignedTx body to /send route.

Throws an error if a user passes both `gasPrice` and `eip1559` params

### CODE REVIEW

File: modules/core/src/v2/coins/eth.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ interface Recipient {
 
 interface SignFinalOptions {
   txPrebuild: {
+    eip1559: { maxPriorityFeePerGas: number; maxFeePerGas: number };
```
Comment (ryan-bitgo) [inline modules/core/src/v2/coins/eth.ts:100]: add question mark

Comment (ryan-bitgo) [reply modules/core/src/v2/coins/eth.ts:100]: ?

File: modules/core/src/v2/wallet.ts [hunk 2]
```diff
@@ -2049,9 +2053,15 @@ export class Wallet {
...
         'instant', 'memo', 'type', 'trustlines', 'transferId',
-        'stakingOptions'
+        'stakingOptions',
       ]);
       const finalTxParams = _.extend({}, halfSignedTransaction, selectParams);
+
+      console.log('/send params');
+      console.log(finalTxParams);
+
+      throw new Error('BAD');
```
Comment (ryan-bitgo) [inline modules/core/src/v2/wallet.ts:2063]: :)

**Comment by karankurbur** [comment]
modules/core/test/v2/unit/wallet.ts was not linted - look for `should error eip1559 and gasPrice are passed`

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**